source/extensions/filters/http/dynamo/dynamo_request_parser.cc

```diff
@@ -91,6 +97,10 @@ RequestParser::TableDescriptor RequestParser::parseTable(const std::string& oper
       }
       return true;
     });
+  } else if (find(TRANSACT_OPERATIONS.begin(), TRANSACT_OPERATIONS.end(), operation) !=
+             TRANSACT_OPERATIONS.end()) {
+    // transactions are always multi-table operations
+    table.is_single_table = false;
```

Comment:
question: even though this is overwhelmingly going to be true (that people use transactions to operate across multiple tables), would it be worth it to literally confirm that here (iterating through the items, similar to how its done above for batch operations)